releng/release-tool/release-tool.go

```diff
@@ -467,6 +482,44 @@ func (r *releaseData) forkProwJobs() error {
 	return nil
 }
 
+// convertRunBeforeMergeToAlwaysRun changes the values in the config for all presubmits from
+// { run_before_merge: true; always_run: false }
+// to
+// { always_run: true }
+func convertRunBeforeMergeToAlwaysRun(configPath string, outputPath string) error {
+	if outputPath == configPath {
+		return fmt.Errorf("output-path and config-path must not be the same")
+	}
+	jobConfig, err := config.ReadJobConfig(configPath)
+	if err != nil {
+		return err
+	}
+	var newPresubmits []config.Presubmit
```

Comment:
don't you want to define this in the loop?

Reply:
Good catch, thank you!